packages/core/base/src/adapter.ts

```diff
@@ -77,18 +92,29 @@ export abstract class BaseWalletAdapter extends EventEmitter<WalletAdapterEvents
     abstract publicKey: PublicKey | null;
     abstract connecting: boolean;
 
+    get supportedTransactionVersions(): Set<TransactionVersion> | null {
+        return null;
+    }
```

Comment:
Making this a getter may make it harder for subclasses to override with a fixed value. I think maybe we can just define it as abstract here, then define the value in `BaseSignerWalletAdapter` and Blocto's adapter.